Sources/UIApplication.swift

```diff
@@ -89,7 +89,7 @@ import JNI
 private let maxFrameRenderTimeInSeconds = 1.0 / 60.0
 
 @_cdecl("Java_org_libsdl_app_SDLActivity_nativeProcessEventsAndRender")
-public func nativeProcessEventsAndRender(env: UnsafeMutablePointer<JNIEnv>, view: JavaObject) {
+public func nativeProcessEventsAndRender(env: UnsafeMutablePointer<JNIEnv?>?, view: JavaObject?) {
```

Comment:
@michaelknoch I also saw some crashes in this function – again these arguments are unused. If this still compiles it may well fix those crashes too